### DISCUSSION TITLE
Nothing after reboot

### DISCUSSION THREAD

**Discussion by Leergzt** [awawa-dev/HyperHDR#911] [Q&A]
First of all, thank you very much for the great project.

However, I have reinstalled my Pi. Installed the latest HyperHDR version, adjusted the config. Configured HyperHDR and everything works perfectly. However, as soon as I turn the Pi off and on again, nothing works. It connects to the router but I can't do anything else. No ping, no SSH, just nothing.

It's probably just something stupid but I can't figure it out. How should I go about solving the problem? I've already re-flashed it.

**Comment by awawa-dev** [comment]
Hi
Dont know probably something related to the OS or SD card... did you configure only HyperHDR or change some OS settings too? Always use clean OS shutdown or configure your system to read-only/overlay mode: https://github.com/awawa-dev/HyperHDR/wiki/Raspberry-Pi-OS-read-only-mode

**Comment by Leergzt** [comment]
Thank you for your help. It was Problem with the pi. Dunno what But i borrowed me a other 3b Pi and it worked…
